### ISSUE TITLE
NTR: 3-methylglutaconyl-CoA hydratase enzyme activity low (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8178]
New term request 3-methylglutaconyl-CoA hydratase enzyme activity low (segal_050209195909): 3-MGCH

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000577
- label: Reduced tissue 3-methylglutaconyl-CoA hydratase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:3082934
- def: Activity of the mitochondrial enzyme 3-methylglutaconyl-CoA hydratase (EC 4.2.1.18) in tissues below the lower limit of normal. The activity can be measured in multiple tissues including culutured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8178
- synonyms:
